dashboard/src/types/tree/TreeDetails.tsx

```diff
@@ -117,3 +121,43 @@ export type TBootsTabData = {
   platformsWithError: string[];
   errorMessageCounts: ErrorMessageCounts;
 };
+
+export type PossibleTabs = Extract<
+  MessagesKey,
+  'treeDetails.builds' | 'treeDetails.boots'
+>;
+
+const possibleTabs = [
+  'treeDetails.builds',
+  'treeDetails.boots',
+] as const satisfies PossibleTabs[];
+
+export const zPossibleValidator = z
+  .enum(possibleTabs)
+  .catch('treeDetails.boots');
+
+const possibleTableFilter = ['error', 'success', 'all'] as const;
```

Comment:
we will need i18n on those

Reply:
this is a state storage, i18n shoul be only in the view